lib/services/import_user.rb

```diff
@@ -0,0 +1,36 @@
+class ImportUser
+  EXPECTED_COLUMNS = %w(first_name last_name email birth_date employment_date password)
+  STATUS_NEW  = 'NEW'
+  STATUS_OK   = 'OK'
+  STATUS_FAIL = 'FAIL'
+
+  attr_reader :error
+
+  def initialize(attributes, options = {})
+    @options = { verbose: false }.merge(options)
+    @attributes = attributes.select { |key| EXPECTED_COLUMNS.include?(key) }
+    @status = 'new'
+    @error = ''
+  end
+
+  def insert_to_db
+    User.create!(@attributes)
+    @status = STATUS_OK
+  rescue StandardError => e
+    @status = STATUS_FAIL
+    @error = e.message
+  end
+
+  def report
+    report_verbose      if      @options[:verbose]
+    report_with_symbol  unless  @options[:verbose]
```

Comment:
@eleidan Maybe it's better to write: `@options[:verbose].present? ? report_verbose : report_with_symbol`  ?